src/main_cli.rs

```diff
@@ -0,0 +1,53 @@
+use std::fs;
+
+use anyhow::{Context, Result};
+use clap::Parser;
+use verifier::handler::{verify, UserError};
+
+#[derive(Parser)]
+#[command(author, version, about, long_about = None)]
+struct Cli {
+    // path to secp256k1 private key file (e.g. /app/secp256k1.sec)
+    #[arg(long)]
+    secp256k1_secret: String,
+
+    // path to secp256k1 public key file (e.g. /app/secp256k1.pub)
+    #[arg(long)]
+    secp256k1_public: String,
```

Comment:
We actually don't need secp256k1 keys. This executable doesn't need to sign response that given attestation is valid. Just return that attestation is valid, with PCRs and enclave key embedded inside the attestation.